bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -109,10 +109,10 @@ metadata:
     capabilities: Full Lifecycle
     categories: AI/Machine Learning, Big Data
     certified: "False"
-    containerImage: quay.io/opendatahub/opendatahub-operator:v2.28.0
-    createdAt: "2025-05-13T09:25:26Z"
+    containerImage: quay.io/opendatahub/opendatahub-operator:v2.29.0
+    createdAt: "2025-05-20T09:33:54Z"
     olm.skipRange: '>=1.0.0 <2.28.0'
```

Comment:
```suggestion
    olm.skipRange: '>=1.0.0 <2.29.0'
```
right?

Reply:
yes! 🤔  why did the automation miss that

Reply:
because it did not have https://github.com/opendatahub-io/opendatahub-operator/pull/1957